liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/DevMojo.java

```diff
@@ -637,9 +637,12 @@ private void injectTestId(Xpp3Dom config) {
         return elements.toArray(new Element[elements.size()]);
     }
 
-    private void runMojo(String plugin, String goal, String serverName, List<String> dependencies)
+    private void runMojo(String groupId, String artifactId, String goal, String serverName, List<String> dependencies)
             throws MojoExecutionException {
-        Plugin mavenPlugin = project.getPlugin(plugin);
+        Plugin mavenPlugin = project.getPlugin(Plugin.constructKey(groupId, artifactId));
+        if (mavenPlugin == null) {
+            mavenPlugin = plugin(groupId(groupId), artifactId(artifactId), version("LATEST"));
```

Comment:
Curious why this is using version LATEST but the [other PR](https://github.com/OpenLiberty/ci.maven/pull/496/files#diff-5489f5e14d778ec413116ee2c13953f4R588) used version RELEASE. Also we will probably have to revise these in the future since both LATEST and RELEASE versions are deprecated.

Reply:
Changed it to do the following:  For Liberty Maven Plugin, we should use LATEST so that it can pickup snapshots and milestones.  For other plugins, we should use RELEASE.